tests/endpoint/common.sh

```diff
@@ -0,0 +1,25 @@
+LEGATO_VERSION="20.04.0"
+
+function download_legato_repo(){
+    echo "Download Legato AF"
+	repo init -u git://github.com/legatoproject/manifest -m legato/releases/${LEGATO_VERSION}.xml
+	repo sync
+}
+
+function build_legato_repo(){
+    if [ ! -d "legato" ]; then
+        echo "Please download the Legato AF first"
```

Comment:
Perform more sanity checks as well. More essential files have to be validated.

Reply:
What else should we check?

Reply:
You should validate the essentials such as `legato-af/Makefile` before proceeding.

Reply:
If you only check the existence of one directory, it might confuse the users who were failing to proceed `repo sync`.